.github/workflows/lint.yml

```
@@ -0,0 +1,21 @@
+name: Lint
+
+on: [push, pull_request]
```

And this option allows us to trigger via a UI button:

```suggestion
on: [push, pull_request, workflow_dispatch]
```